packages/frontend/src/components/common/MigrationBanner.js

```diff
@@ -0,0 +1,160 @@
+    background-color: #FFECD6;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    a {
+        color: #995200;
+        text-decoration: underline;
+        cursor: pointer;
+
+        :hover {
+            color: #995200;
+        }
+    }
+`;
+
+const ContentWrapper =  styled(Container)`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 0;
+
+    &>*:first-child{
+        margin-right: 10px;
+    }
+
+    @media (max-width: 992px) {
+        padding: 16px;
+    }
+
+    @media (max-width: 768px) {
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 16px;
+    }
+
+    & .content {
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: none;
+
+        svg{
+            margin-right: 18px;
+            min-width: 24px;
+        }
+    }
+`;
+
+const CustomButton = styled(FormButton)`
+    color: #452500 !important;
+    background-color: #FFB259 !important;
+    border: none !important;
+    white-space: nowrap;
+    padding: 11px 16px;
+    margin: 0 !important;
+    height: 40px;
+
+    @media (max-width: 768px) {
+        margin-top: 16px !important;
+    }
+`;
+
+const MigrationBanner = ({ account }) => {
+    const migrationStartDate = new Date( MIGRATION_START_DATE * 1000);
+    const migrationEndDate = new Date(MIGRATION_END_DATE * 1000);
+
+    const showPostMigrationBanner = isAfter(new Date(), new Date(migrationStartDate));
+
+
+    const setBannerHeight = () => {
+        const migrationBanner = document.getElementById('migration-banner');
+        const bannerHeight = migrationBanner ? migrationBanner.offsetHeight : 0;
+        const networkBanner = document.getElementById('top-banner');
+        if (networkBanner){
+            networkBanner.style.top = bannerHeight ? `${bannerHeight}px` : 0;
+        } else {
+            const app = document.getElementById('app-container');
+            const navContainer = document.getElementById('nav-container');
+            navContainer.style.top = bannerHeight ? `${bannerHeight}px` : 0;
+            app.style.paddingTop = bannerHeight ? `${bannerHeight + 85}px` : '75px';
+        }
+    };
+
+
+    const getInviteCalendarEventURl =()=>{
+        const calenderURL = new URL('https://calendar.google.com/calendar/u/0/r/eventedit');
+        const calendarStartDate= migrationStartDate.toISOString().replace(/-|:|\.\d\d\d/g,'');
+        const calendarEndDate= migrationEndDate.toISOString().replace(/-|:|\.\d\d\d/g,'');
+
+        calenderURL.searchParams.append('dates', `${calendarStartDate}/${calendarEndDate}`);
+        calenderURL.searchParams.append('text', 'Wallet Migration Event');
+        calenderURL.searchParams.append('details', 'This is the official start date of the near wallet migration. Please make sure to start migrating your account as soon as possible.');
```

Comment:
We need to set these up for localization

Reply:
Sure thing! Thanks.